cpp/src/Slice/FileTracker.h

```diff
@@ -3,6 +3,7 @@
 #ifndef FILE_TRACKER_H
 #define FILE_TRACKER_H
 
+#include "Ice/LocalException.h"
```

Comment:
Is this needed?

Reply:
Yes, we define a `FileException` which inherits from `LocalException`.
We used to get this include transitively because `Parser.h` included it, but now it doesn't.